build.psm1

```diff
@@ -543,6 +543,13 @@ Fix steps:
         }
     }
 
+    if ($Environment.IsUbuntu -or $Environment.IsDebian)
+    {
+        # psrp 1.3* now depends on libgssapi_krb5 for Kerberos support.
+        # selected Ubuntu and Debian systems do not have a link for it.
```

Comment:
> selected Ubuntu

what is the "selected" ubuntu and debian system? Do you mean all ubuntu/debian distros we officially support? Can you please update the comment to make it a bit clear?